linera-service/src/cli_wrappers/wallet.rs

```diff
@@ -213,17 +237,14 @@ impl ClientWrapper {
         .await
     }
 
-    /// Returns an iterator over the arguments that should be added to all command invocations.
-    fn command_arguments(&self) -> impl Iterator<Item = Cow<'_, str>> + '_ {
+    fn required_command_arguments(&self) -> impl Iterator<Item = Cow<'_, str>> + '_ {
         [
             "--wallet".into(),
             self.wallet.as_str().into(),
             "--keystore".into(),
             self.keystore.as_str().into(),
             "--storage".into(),
             self.storage.as_str().into(),
-            "--max-pending-message-bundles".into(),
-            self.max_pending_message_bundles.to_string().into(),
```

Comment:
Are we even setting that in any tests?

This selection of arguments looks a bit arbitrary to me.